data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -276,12 +276,13 @@ def _load_downloadable_zip(
             inplace=False,
         )
 
-        logger.info("Writing downloadable csv")
-        downloadable_df_copy.to_csv(csv_path, index=False)
-
         logger.info("Writing downloadable excel")
         downloadable_df_copy.to_excel(excel_path, index=False)
 
+        logger.info("Writing downloadable csv")
+        downloadable_df_copy[self.GEOID_TRACT_FIELD_NAME] = "\"" + downloadable_df_copy[self.GEOID_TRACT_FIELD_NAME] + "\""
+        downloadable_df_copy.to_csv(csv_path, index=False)
```

Comment:
instead of manually adding the quotes, we could also add this to the `to_csv`: 

`downloadable_df_copy.to_csv(.... quoting=csv.QUOTE_NONNUMERIC)` See https://docs.python.org/3/library/csv.html 

But manually adding is also fine!

Reply:
Oh cool, I didn't know about that! I'll update it.

Reply:
Actually couldn't get this to work in 15 minutes, so I'm going to keep as is (doesn't seem worth continuing to troubleshoot or doing a bigger refactor to how we invoke CSV write).

Reply:
Good call!